AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialCanvas_Atom_BasicTests.py

```diff
@@ -0,0 +1,135 @@
+    from editor_python_test_tools.utils import Report, Tracer, TestHelper
+
+    with Tracer() as error_tracer:
+        # Set constants before starting test steps.
+        test_1_material_graph = os.path.join(atom_tools_utils.MATERIALCANVAS_GRAPH_PATH, "test1.materialgraph")
+        test_2_material_graph = os.path.join(atom_tools_utils.MATERIALCANVAS_GRAPH_PATH, "test2.materialgraph")
+        test_3_material_graph = os.path.join(atom_tools_utils.MATERIALCANVAS_GRAPH_PATH, "test3.materialgraph")
+        test_4_material_graph = os.path.join(atom_tools_utils.MATERIALCANVAS_GRAPH_PATH, "test4.materialgraph")
+        test_5_material_graph = os.path.join(atom_tools_utils.MATERIALCANVAS_GRAPH_PATH, "test5.materialgraph")
+
+        # 1. Open an existing material graph document.
+        material_document_id = atom_tools_utils.open_document(test_1_material_graph)
+        Report.result(
+            Tests.open_existing_material_graph,
+            atom_tools_utils.is_document_open(material_document_id) is True)
+
+        # 2. Close the selected material graph document.
+        atom_tools_utils.close_document(material_document_id)
+        Report.result(
+            Tests.close_opened_material_graph,
+            atom_tools_utils.is_document_open(material_document_id) is False)
+
+        # 3. Open multiple material graph documents then use the CloseAllDocuments bus call to close them all.
+        for material_graph_document in [test_1_material_graph, test_2_material_graph, test_3_material_graph,
+                                        test_4_material_graph, test_5_material_graph]:
+            atom_tools_utils.open_document(os.path.join(material_graph_document))
+        Report.result(
+            Tests.close_all_opened_material_graphs,
+            atom_tools_utils.close_all_documents() is True)
+
+        # 4. Open multiple material graph documents then verify all material documents are opened.
+        material_graph_1_document_id = atom_tools_utils.open_document(os.path.join(test_1_material_graph))
+        material_graph_2_document_id = atom_tools_utils.open_document(os.path.join(test_2_material_graph))
+        material_graph_3_document_id = atom_tools_utils.open_document(os.path.join(test_3_material_graph))
+        material_graph_4_document_id = atom_tools_utils.open_document(os.path.join(test_4_material_graph))
+        material_graph_5_document_id = atom_tools_utils.open_document(os.path.join(test_5_material_graph))
+        Report.result(
+            Tests.verify_all_material_graphs_are_opened,
+            atom_tools_utils.is_document_open(material_graph_1_document_id) is True and
+            atom_tools_utils.is_document_open(material_graph_2_document_id) is True and
+            atom_tools_utils.is_document_open(material_graph_3_document_id) is True and
+            atom_tools_utils.is_document_open(material_graph_4_document_id) is True and
+            atom_tools_utils.is_document_open(material_graph_5_document_id) is True)
```

Comment:
DRY: That's a lot of repeated code here to repeat the same behavior.

Additionally, is the `os.path.join()` needed here since all of those test variables were generated using `os.path.join`?

Reply:
removed `os.path.join()` was just a simple mistake and reduced DRY by adding a `for` loop - thanks for the feedback